examples/relevancy-cli/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "examples-relevancy-cli"
+version = "0.1.0"
+authors = ["sync-team@mozilla.com"]
```

Comment:
Homesick? ;)

Reply:
Oops, copy and paste error.  I think this is an optional field so I'm just going to remove it.